src/main/java/mate/academy/dao/OrderDao.java

```diff
@@ -0,0 +1,16 @@
+package mate.academy.dao;
+
+import java.util.List;
+import java.util.Optional;
+import mate.academy.model.Order;
+import mate.academy.model.User;
+
+public interface OrderDao {

```

Comment:
Minor comment, you actually use only add and getByUser methods, so it's not necessary to implement 
`Optional<Order> getById(Long id) and List<Order> getAll() `methods